WordPress/Classes/ViewRelated/Blaze/Webview/BlazeFlowCoordinator.swift

```diff
@@ -23,8 +23,9 @@ import UIKit
         }
     }
 
+    /// Blaze overlays should be shown once only, regardless of entry point
     var key: String {
-        return description
+        return ""
```

Comment:
@kean I'm not sure what to name this key. I'm debating whether or not to return dashboard_card for the source key, so that users who've already seen won't see it again. But on the other hand I don't think it's terrible to show the overlay again. Lmk what you think!

Reply:
Returning `dashboard_card` seems a like a good idea, assuming that's most users either clicked the dashboard card or haven't tried Blaze at all. I would just drop a comment to clarify why it's returning it.

<hr/>

(nit) The `OverlayFrequencyTracker` API is a bit confusing. Both the `OverlaySource` and its `key` are non-optional, yet technically they are: you can pass an empty string as a key, and it'll work fine. The main parameter is `OverlayType`, but it's the second in the initializer.

It could be simplified to just (no `OverlaySource`; frequency is a `FrequencyConfig`):

```
// Remove `OverlaySource` and `OverlayType` – let the user construct the unique identifier based on their requirements.
// `FrequencyConfig` should probably be an enum.
OverlayFrequencyTracker(identifier: "blaze", frequency: .showOnce)
```

Reply:
> Returning dashboard_card seems a like a good idea, assuming that's most users either clicked the dashboard card or haven't tried Blaze at all. I would just drop a comment to clarify why it's returning it.

Done! [56c2996](https://github.com/wordpress-mobile/WordPress-iOS/pull/21125/commits/56c2996cd6d3d55f2d390194baddc5aee22f18a1)

> The OverlayFrequencyTracker API is a bit confusing.

I'll look into this in a different PR